src/semantize/typing.nit

```diff
@@ -873,7 +877,7 @@ end
 
 redef class APropdef
 	# The entry point of the whole typing analysis
-	fun do_typing(modelbuilder: ModelBuilder)
+	fun do_typing(modelbuilder: ModelBuilder, disable_warning: nullable Bool)
```

Comment:
Maybe pass the `TypeVisitor` instance instead?

```suggestion
	fun do_typing(v: TypeVisitor)
```

Then you can access `modelbuilder` and `disable_warning` (or better, the `warn` method :))